proto/catalog.proto

```diff
@@ -92,8 +92,9 @@ message Sink {
 message Connection {
   message PrivateLinkService {
     string provider = 1;
-    string endpoint_id = 2;
-    map<string, string> dns_entries = 3;
+    string service_name = 2;
+    string endpoint_id = 3;
+    map<string, string> dns_entries = 4;
```

Comment:
The privatelink proto interface is still private. These changes would not break users, any work around to fix these checks? @BugenZhao

Reply:
The check is not required and we can ignore it.